browser-test/browser-test-compose.yml

```diff
@@ -31,4 +31,5 @@ services:
       - DB_JDBC_STRING=jdbc:postgresql://database:5432/postgres
       - BASE_URL=http://civiform:9000
       - CIVIFORM_TIME_ZONE_ID
+      - CIVIFORM_API_KEYS_BAN_GLOBAL_SUBNET=false
```

Comment:
Not blocking: For consistency, would it make sense to add this to the other compose files (e.g. `bin/run-dev` and `bin/sbt-test`)?

If so, I'd consider adding this to `application.dev.conf` instead.

Reply:
The default is to not allow it, and I'm inclined to leave it that way. I'm overriding it for browser tests because we don't know what IP address it will come from

Reply:
Sounds good. Given that, I'd add a comment stating that for posterity.